### PR TITLE
feat(core): changes to publish the library as web component

### LINKED ISSUE #16: Release the workflow creator as a web component
**Is your feature request related to a problem? Please describe.**
Currently workflow creator is available as an angular library 
but this restricts us from applying it in other frontend frameworks like react, vue etc.

**Describe the solution you'd like**
Should be available as a web component/angular element

### PR DESCRIPTION
**Pull request by yeshamavani**
GH-16

## Description

Changes to publish the library as web component
added lerna to publish multiple packages at simultaneously

Fixes #16

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        run: |
+          git config --global user.name '@yeshamavani'
+          git config --global user.email 'yesha.mavani@sourcefuse.com'
+          git remote set-url origin https://$GITHUB_ACTOR:$GITHUB_PAT@github.com/sourcefuse/workflows-creator
+          GITHUB_PAT: ${{ secrets.RELEASE_COMMIT_GH_PAT }}
+      - name: Authenticate with Registry
+        run: |
+          echo "@shubhamp-sf:registry=https://registry.npmjs.org/" > .npmrc
+          echo "registry=https://registry.npmjs.org/" >> .npmrc
+          echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> .npmrc
```
Comment (akshatdubeysf) [inline .github/workflows/release.yml:29]: why are we echoing npm token?

File: projects/workflows-creator/package.json [hunk 2]
```diff
@@ -62,40 +62,5 @@
     "registry": "https://registry.npmjs.org/",
     "access": "public",
     "directory": "dist"
-  },
-  "release": {
-    "branches": [
-      "master"
-    ],
-    "plugins": [
```
Comment (akshatdubeysf) [inline projects/workflows-creator/package.json:70]: where are we moving the release config to?

**Review by akshatdubeysf — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (yeshamavani) [reply .github/workflows/release.yml:29]: will remove that

File: projects/workflows-creator/package.json (hunk 2 above)
Comment (yeshamavani) [reply projects/workflows-creator/package.json:70]: this config was needed when we publish via semantic 
now to publish both packages simultaneously will use lerna for that.
So not needed

**Review by shubhamp-sf — APPROVED** [review]
LGTM

**Pull request merged**